openfisca_us/variables/household/demographic/marital_unit/marital_unit_weight.py

```diff
@@ -0,0 +1,13 @@
+from openfisca_us.model_api import *
+
+
+class marital_unit_weight(Variable):
+    value_type = float
+    entity = MaritalUnit
+    label = "Marital unit weight"
+    definition_period = YEAR
+
+    def formula(marital_unit, period, parameters):
+        # If no weight provided, use the average of the people in the marital unit.
```

Comment:
How about we use household weight instead? This is ultimately what we'll do for everything after reweighting. I'm not sure this approach will yield the total number of marital units.